webserver/views/index.py

```diff
@@ -41,3 +45,33 @@ def faq():
 @index_bp.route("/roadmap")
 def roadmap():
     return render_template("index/roadmap.html")
+
+
+# listen-group    listens                        0   282202115       282202115       0               none
+@index_bp.route("/current-status")
+def current_status():
+
+    load = "%.2f %.2f %.2f" % os.getloadavg()
+
+    process = subprocess.Popen([config.KAFKA_RUN_CLASS_BINARY, "kafka.tools.ConsumerOffsetChecker",
```

Comment:
I spent 2 hours looking for a way and then you beat me to it by finding the command line program. At this point I feel that:

1. this isn't going to be a popular page, so it isn't going to be loaded often.
2. the VM will already be resident in ram and we have plenty of ram still.
3. If we find a way to do this via the API, then I'm happy to move to it.

In the meantime, I think we should deploy this to get insights into our server going.